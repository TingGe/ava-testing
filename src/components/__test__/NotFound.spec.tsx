import * as Enzyme from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import * as React from 'react';
import NotFound from '../NotFound';

Enzyme.configure({ adapter: new Adapter() });

describe('NotFound', () => {
  test('NotFound render', () => {
    const wrapper = Enzyme.shallow(<NotFound />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
