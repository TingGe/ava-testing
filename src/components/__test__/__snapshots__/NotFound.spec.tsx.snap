// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound NotFound render 1`] = `
<span>
  404 Not Found
</span>
`;
